pages/data-design/multiple-sources-on-avo-branches.mdx

```diff
@@ -1,7 +1,6 @@
 ---
 layout: 'guide'
 title: 'Multiple sources on Avo branches'
-# abstract: 'Keep your Avo main branch in sync with your git main branch'
```

Comment:
Did we want to include these abstracts, or intentionally wiping them away?

Reply:
On most I removed the comment but some were copy-pastes where the abstract didn't match the title, like this one, and those got removed.